src/Board.coffee

```diff
@@ -61,6 +61,8 @@ class Board
     COLUMNS: BOARD_COLS
     WIDTH: BOARD_WIDTH
     HEIGHT: BOARD_HEIGHT
+  # HERE -> thinking about adding click disable flag...
+  @pauseCollectCoins = false
```

Comment:
you have defined a class property here however you are using it as if it were an instance property. Which is not valid.

You should use `@pauseCollectCoins` in the constructor of the class to define an instance property.

Effectively this line of code here has zero effect on the functionality of your code the way it is written.

To use the class property, you would refer to it as `Board.pauseCollectCoins`